src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -213,15 +211,12 @@ private static ImmutableArray<ReferenceDirective> GetReferenceDirectives(Compila
             IList<ReferenceDirectiveTriviaSyntax> directiveNodes = compilationUnit.GetReferenceDirectives(
                 d => !d.File.ContainsDiagnostics && !string.IsNullOrEmpty(d.File.ValueText));
             if (directiveNodes.Count == 0)
-            {
                 return ImmutableArray<ReferenceDirective>.Empty;
-            }
 
             var directives = ArrayBuilder<ReferenceDirective>.GetInstance(directiveNodes.Count);
             foreach (var directiveNode in directiveNodes)
```

Comment:
>foreach (var directiveNode in directiveNodes) [](http://example.com/codeflow?start=12&length=45)

I don't think we do this for `foreach`. Could be wrong, but I never encountered one